Lesson_12Hom/contact.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!-- <link rel="icon" href="#" type="images/x-icon"> -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,700;0,900;1,300;1,400;1,700;1,900&family=Poltawski+Nowy:ital,wght@0,500;0,600;1,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/general.css">
+    <link rel="stylesheet" href="styles/contact.css">
+    <title>Parettool Laser</title>
+</head>
+<body>
+    <header>
+        <div class="cap">
+            <div class="adress">
+                <p>м. Київ</p>
+                <p>вул. Полярна, 3</p>
+            </div>
+            <img src="img/logo-paretool.png" alt="Logo Parettool">
+            <div class="telephone">
+                <p>+38-063-344-17-36</p>
+                <p>+38-099-159-79-57</p>
+            </div>
+        </div>
+        <div class="nav">
+            <h1>Послуги нанесення лазерного гравірування на металах</h1>
+            <nav>
+                <a href="index.html">Номе</a>
+                <a href="#">Галерея</a>
+                <a href="#">Новини</a>
+                <a href="contact.html">Контакти</a>
+            </nav>
+        </div>
+    </header>
+    <div class="cont">
+        <h2>Контакти</h2>
+        <div>
+            <span>Телефон:</span>
+            <p>+38-063-344-17-36, +38-099-159-79-57</p>
+        </div>  
+        <div>
+            <span>Адреса:</span>
+            <p>м. Київ, вул. Полярна, 3</p>
+        </div>  
+        <div class="adress-map">
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2536.7314416047466!2d30.44981067689995!3d50.52056358231251!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4d28f6961cdeb%3A0x2a1b2810b014a045!2z0YPQuy4g0J_QvtC70Y_RgNC90LDRjywgMywg0JrQuNC10LIsIDAyMDAw!5e0!3m2!1sru!2sua!4v1686158050232!5m2!1sru!2sua" width="700" height="300" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+        </div> 
+    </div>
+    <div class="contact-form">
+        <label for="first-name">Ім'я</label>
+        <input type="text" name="first-name" id="first-name" placeholder="Тарас" >
```

Comment:
placeholder it's more about duplicating label, not examples of names

It's not a problem and not a mistake, but generally you will it very rare in placeholders